src/domain/customers/utils.ts

```diff
@@ -2,29 +2,22 @@ import { CUSTOMERS } from './__generated__/CUSTOMERS';
 
 export interface CustomerData {
   id: string;
-  firstName: string;
-  lastName: string;
-  nickname: string | null;
-  email: string | false | null;
-  phone: string | false | null;
+  name: string;
```

Comment:
Merging firstName and lastName fields here is fine, but it doesn't make much sense to have a seperate nickname field after that.